### ISSUE TITLE
[RA2][RC2] Add sig-ui testing in RC2

### ISSUE THREAD

**Issue by collivier** [anuket-project/anuket-specifications#2321]
RA1/RC1 asks to run basic horizon testing.
To make the overall testing coherent, we must set sig-ui testing as mandatory.

**Comment by CsatariGergely** [comment]
@collivier do we need any specific requirements to be added to RA2 for this?

**Comment by rgstori** [comment]
agreed to keep k8s dashboard (and related tests) as optional for ra2 and rc2

**Comment by rgstori** [comment]
@collivier is this still relevant? not all caas distributions come with the vanilla k8s dashboard, or any dashboard at all
